server.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import unicode_literals
+import socket
+
+
+_CRLF = b'\r\n'
+_GET = b'GET'
+_PROTOCOL = b'HTTP/1.1'
+_HOST_PREFIX = b'Host:'
+_WS = b' '
+
+_RESPONSE_TEMPLATE = _CRLF.join([
+    b'HTTP/1.1 {response_code} {response_reason}',
+    b'Content-Type: text/html; charset=UTF-8',
+    b''])
+
+
+def response_ok():
+    return _RESPONSE_TEMPLATE.format(response_code=b'200',
+                                     response_reason=b'OK')
+
+
+def response_error(response_code, response_reason):
+    """ Return 500 Internal Server Error"""
+    return _RESPONSE_TEMPLATE.format(response_code=response_code,
+                                     response_reason=response_reason)
+
+
+def parse_request(request):
+    lines = request.split(_CRLF)
```

Comment:
This is a dangerous assumption.  Better to first split the head from the body and then only treat the head as "lines"